solutionbox/structured_data/mltoolbox/_structured_data/_package.py

```diff
@@ -125,8 +125,17 @@ def _wait_and_kill(pid_to_wait, pids_to_kill):
 # Analyze
 # ==============================================================================
 
-
 def analyze(output_dir, dataset, cloud=False, project_id=None):
+  """Blocking version of analyze_async. See documentation of analyze_async."""
+  job = analyze_async(
+      output_dir=output_dir,
+      dataset=dataset,
+      cloud=cloud,
+      project_id=project_id)
+  job.wait()
+  return job
```

Comment:
We don't need to return anything here. The job object is not useful once the job is completed.